tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py

```diff
@@ -249,7 +252,7 @@ def verify(self, pv):
                               MODE_TLV,
                               ADDRESS_REGISTRATION_TLV
                              } < set(p.mle.tlv.type) and\
-                   len(p.mle.tlv.addr_reg_iid) > 0 and\
+                   len(p.mle.tlv.addr_reg_iid) == 2 and\
```

Comment:
```suggestion
                   len(p.mle.tlv.addr_reg_iid) >= 2 and\
```

Reply:
If test with Thread 1.2 build with DUA feature enabled, DUA would always be registered closely after MLEID. That is, there is chance that extra addresses might be registered as well, better to relax the checks.

Reply:
sure, modified